### DISCUSSION TITLE
Вернуть тему "Предложения" на форуме

### DISCUSSION THREAD

**Discussion by SamonJeff** [octothorp-team/public-dobrograd#925] [Предложения]
До сих пор не могу понять адекватную причину по которой перенесли все идей и предложений тут. Их почти никто не читает включая игроков и людей которые отвечают на них, 90% из обсуждений весят без ответа уже как минимум 4 месяца (я даже не удивлюсь если про эту тему тоже забудут). Мнение игроков сообщество является очень важным, если проигнорировать их и делать все самому - возникнут проблемы. Был бы рад услышать ваш ответ, а не через лайки и дизлайки в канале #commits. Всем спасибо. (https://github.com/octothorp-team/public-dobrograd/discussions/925)
![image_2022-02-07_172303](https://user-images.githubusercontent.com/99187919/152796072-c814509d-a7d3-4cea-a4a7-49ca314437b2.png)

**Comment by inquizzyoff** [comment]
Уже сделали 😎
